frontend/src/features/admin-form/create/logic/components/EmptyLogic.tsx

```diff
@@ -24,27 +37,31 @@ export const EmptyLogic = (): JSX.Element => {
       color="secondary.500"
       pt={{ base: '0.5rem', md: '2.75rem' }}
     >
-      <Text textStyle="h2" as="h2">
-        Start creating logic for your form
-      </Text>
-      <Text textStyle="body-1" mt="1rem">
-        Show or hide fields depending on user input, or disable form submission
-        for invalid answers.{' '}
-        <Link isExternal href={GUIDE_FORM_LOGIC}>
-          Learn to work with logic
-        </Link>
-      </Text>
-      <Button
-        my="2.5rem"
-        leftIcon={<BiPlus fontSize="1.5rem" />}
-        onClick={setToCreating}
-      >
-        Add logic
-      </Button>
-      <LogicSvgr maxW="292px" />
-      <Divider my="2.5rem" />
+      <Skeleton isLoaded={isLoaded}>
+        <Text textStyle="h2" as="h2">
+          Start creating logic for your form
+        </Text>
+      </Skeleton>
+      <Skeleton isLoaded={isLoaded}>
```

Comment:
imo loading state should be modelled after a "populated" state, meaning we might want to create an entirely separate loading state similar to admin form builder and/or public form page

Reply:
Same as my reply to anty's comment :P

Reply:
if its figma then its figma~ let designer review then